jenkins_jobs/trigger_weekly_host_os_build/script.sh

```diff
@@ -115,6 +118,21 @@ create_symlinks() {
           "${UPLOAD_SERVER_USER_NAME}@${UPLOAD_SERVER_HOST_NAME}:${UPLOAD_SERVER_WEEKLY_DIR}/"
 }
 
+tag_git_repos() {
+    local repos_push_urls=$@
+    local version_file="$(basename $VERSIONS_PUSH_URL .git)/VERSION"
```

Comment:
The default versions repository location is no longer `versions`, it's `workspace/repositories/versions`.
Also, you have the repo name in a var, you don't need the `basename` command.